main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,177 @@
+`yarn –version`
 
-```sh secondary style2
-# Terminal 2
-cd demo
-agoric open --repl
+![049](./assets/049.png)
+
+
+# Installing Docker
+Now you'll install Docker using the two commands below. This first command will add the Docker GPG keys to your system, then add the repository to Apt for installation.
+```
+# Install Docker
+# Add Docker's official GPG key:
+sudo apt-get update
+sudo apt-get install ca-certificates curl gnupg
+sudo install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+sudo chmod a+r /etc/apt/keyrings/docker.gpg
+
+# Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt-get update
 ```
+![027](./assets/027.png)
 
-This should automatically open [http://127.0.0.1:8000](http://127.0.0.1:8000) in a new browser window or tab.
+Now you can install Docker!
+`sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin`
 
-To begin using the wallet, click the "Connect Solo Wallet" button.
+![028](./assets/028.png)
 
-![agoric wallet solo connection](./assets/agoric-open-repl-1.png)
+Now that Docker has been installed you'll need to add your user account to the Docker group.
+`sudo usermod -aG docker $USER`
 
-After your solo wallet is connected, then you're ready to deploy the contract and API. 
+![029](./assets/029.png)
 
-## Deploy the Contract and API
+Now go ahead and reboot your image.
 
-In our second terminal, deploy the contract to the simulated blockchain
-and the API to the solo client.
+![030](./assets/030.png)
 
-```sh secondary style2
-# Terminal 2
-cd demo # if not already there
-agoric deploy ./contract/deploy.js 
-agoric deploy ./api/deploy.js
-```
+Once your image has rebooted and you've logged back on, test that Docker works by running the `hello-world` sample.
+`docker run hello-world`
 
-We'll cover [deploying smart contracts](./deploying.md)
-in detail later.
+![031](./assets/031.png)
 
-## Start the Dapp User Interface
 
-The web user interface communicates with the API in the solo client as well as the wallet.
+# Installing the Sample Dapp
+Now you'll use yarn to pull down the sample dapp. The sample dapp will be placed in a subfolder named `demo`.
+`yarn create @agoric/dapp --dapp-base https://github.com/agoric-labs/ --dapp-template dapp-game-places demo`
 
-```sh secondary style3
-# Terminal 3
-cd demo # if not already there
-cd ui && yarn start
-```
+![024](./assets/024.png)
+
+
+# Install Dapp Dependencies
+Now navigate to the `demo` directory and run the `yarn install` command to install any solution dependencies.
+`cd demo`
+`yarn install`
+
+![033](./assets/033.png)
+
+
+# Starting the Network
```

Comment:
"the Network"? any phrase of the form "the X" assumes the reader knows what X is.

```suggestion
## Starting a local Agoric blockchain
```

That's what I used in #881 . Well, I used "Start" rather than "Starting".

Reply:
changed